power_cell_detector.py

```diff
@@ -0,0 +1,55 @@
+import cv2
+import numpy as np
+# from depth_live_generator import DepthLiveGenerator
+
+
+class PowerCellDetector:
+
+    # Constructor
+    def __init__(self, generator):
+        self.input = generator
+        self.LOW_YELLOW = np.array([17, 120, 50])
+        self.HIGH_YELLOW = np.array([33, 255, 250])
+
+    # Runs the detector. This means that it'll actually detect the loading bay
+    # and then it'll calibrate to the loading bay green color. This method returns
+    # the contours of the actual loading bay.
+
+    def run_detector(self):
+        img, depth_frame = self.input.generate()
+
+        if img is None:
+            return None, None
+
+        output = img.copy()
+        hsv = cv2.cvtColor(img, cv2.COLOR_BGR2HSV)
+        hsv = cv2.medianBlur(hsv, 5)
+        mask = cv2.inRange(hsv, self.LOW_YELLOW, self.HIGH_YELLOW)
+        mask = cv2.erode(mask, None, iterations=7)
```

Comment:
Again, what's going on with the erode/dilate? Can we get some comments on what each step does?